README.md

```diff
@@ -144,22 +144,24 @@ Build the required projects
 sbt standalone/assembly cli/assembly gateway/assembly
 ```
 
-First set up the dataset. This should create the keyspaces and tables in Cassandra. 
+First initialize the dataset. This should create the keyspaces and tables in Cassandra. 
```

Comment:
There is no more dataset to initialize.  Maybe you mean "First initialize the C* keyspaces" ?

Reply:
That is right. This is already clarified in the following sentence.